build.gradle

```diff
@@ -0,0 +1,137 @@
+// TODO(bdrutu): Default build directory is build but on MacOs with a case-insensitive
+// File System would conflict with Bazel BUILD file.  Remove this and update the
+// .gitignore entry once Bazel is removed.
+buildDir = 'build_gradle'
```

Comment:
Removed. Not a problem here because we do not build anything in the root directory that can conflict with the BAZEL file.